docs/api/standard_library_types.md

```diff
@@ -56,16 +56,17 @@ Pydantic supports the following [datetime](https://docs.python.org/library/datet
 types:
 
 ### [`datetime.datetime`][]
-* `datetime` fields will accept values of type:
 
-    * `datetime`; an existing `datetime` object
-    * `int` or `float`; assumed as Unix time, i.e. seconds (if >= `-2e10` and <= `2e10`) or milliseconds
+- `datetime` fields will accept values of type:
+
```

Comment:
The added newline broke the rendering. Was it added by the linter? It seems the extra level of nesting below did not have the same newline added.

| Before                                                                                    | After                                                                                     |
|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
| ![image](https://github.com/user-attachments/assets/643d5681-a822-48c8-85a9-445f25f16156) | ![image](https://github.com/user-attachments/assets/21435ae0-5eb9-47c6-9c4f-419e82811729) |

Reply:
As discussed below this is due to the 2 vs. 4 space indentation. 4 spaces indentation is required by `python-markdown`.